src/lib/indexer/index.ts

```diff
@@ -1,27 +1,37 @@
 import { Logger } from 'pino'
 
-import type Database from '../db/index.js'
-import ChainNode from '../chainNode.js'
+import type Database from '../db/index'
+import ChainNode from '../chainNode'
+
+export type ChangeSet = Record<string, never>
```

Comment:
Hopefully it's obvious but the intent is to update this type to include the different types of upserts that will need to be made agains tthe database. Blocks will then be processed into a set of changes that can then all be applied in a single tx